profiles/base.yaml

```diff
@@ -893,6 +893,10 @@ workorder_cash_on_hand:
 retain_crafting_materials: false
 # Whether you want to repair your crafting tools after completing workorders or not. For those with tools setup under gear, set to false in your yaml.
 workorders_repair: true
+workorders_materials:
```

Comment:
This setting is named to be specific to `workorders`, but the overridden things (`sew-stock-number`, etc.) are also used elsewhere (e.g. `trade`). Would other scripts want this functionality?

Reply:
Trade uses workorders, so this change gives that functionality to Trade. I think Craft would have to be rewritten with new ranges in order to use this materials adjustment. I don't use Craft, but if someone who does wants this, I'm all ears on a tactic for that.

Workorders is kind of a special case for this kind of change, in that it has a set number of items it's going to do. This makes changing the materials a niche adjustment. Craft, for example, can just do 2.5 items and be efficient (this is why some prefer it, that don't care about prestige).